cmds/ls/fileinfo_linux.go

```diff
@@ -106,14 +107,33 @@ type quotedStringer struct {
 }
 type longStringer struct {
 	fileInfo
-	comp fmt.Stringer // decorator pattern
+	comp  fmt.Stringer // decorator pattern
+	human bool
 }
 
 // Return the name surrounded by quotes with escaped control characters.
 func (fi quotedStringer) String() string {
 	return fmt.Sprintf("%#v", fi.name)
 }
 
+func sizeToString(size int64) string {
+	switch {
```

Comment:
This needs to be in a package. I am not that concerned about what package, but having it special in ls is taking us down a bad path.